### ISSUE TITLE
Admin CP Version issues

### ISSUE THREAD

**Issue by AetherUnbound** [rafalp/Misago#1305]
Related to #1291, #1292, and #1294 

It seems the changes I implemented are not rendering the response correctly:
![image](https://user-images.githubusercontent.com/10214785/66249727-a1a31280-e6ec-11e9-8295-e9209be62885.png)

**Comment by AetherUnbound** [comment]
Update: I cleared the entry from my redis cache (it was still there?) and then the check ran successfully. This must have been due to a change in what was being cached. It looks like previously, the _entire message_ was cached, whereas now we're only caching the version. 
https://github.com/rafalp/Misago/pull/1292/files#diff-d63ed61ba084e2f4d9bd1b5e959a886dL21-L26
I think that was intentional, since we only wanted to cache the PyPi version now anyway. Does that sound right to you @rafalp? If so, I guess we can close this ticket. It was just weird I had to flush redis myself, maybe there should be an `./appctl` command for flushing all entries in redis?
